tools/msvs/install_tools/install_tools.bat

```diff
@@ -47,7 +47,23 @@ echo script is at your own risk. Please read the Chocolatey's legal terms of use
 echo and the Boxstarter project license as well as how the community repository
 echo for Chocolatey.org is maintained.
 echo.
-echo You can close this window to stop now.
+pause
+
+cls
+echo !!!!!WARNING!!!!!
+echo -----------------
+echo Use of Boxstarter may reboot your box automatically multiple times. When
+echo performing a reboot, Boxstarter will need to disable UAC to allow the
```

Comment:
Is it worth it to spell out what _UAC_ refers to? Or are most Windows users going to know?

Reply:
As a developer using Windows, I guess you should know what it means. Looks much better now.

Reply:
I see this was fixed up by @joaocgreis